builder-server/src/server.rs

```diff
@@ -93,14 +101,309 @@ where
     I: AsRef<A> + Send + Sync,
     A: Builder<E>,
 {
+    let content_type_header = headers.get(ACCEPT);
+    let content_type_str = content_type_header
+        .and_then(|value| value.to_str().ok())
+        .unwrap_or("application/json");
+    let content_type = match Accept::from_str(content_type_str) {
+        Ok(Accept::Ssz) => {
+            info!("REQUESTED SSZ");
```

Comment:
are these relics of debugging?

Reply:
Yeah, can remove now that we have tested it out reasonably well